src/base/preferences.cpp

```diff
@@ -53,9 +55,9 @@
 
 #include "utils/fs.h"
 #include "utils/misc.h"
```

Comment:
Once you have decided to sort it, then do it right. Headers should be in alphabetical order within each group so move "logger.h" and "settingsstorage.h" on top of "utils/fs.h".